src/PowerShellEditorServices/Utility/PsesLogger.cs

```diff
@@ -101,6 +106,28 @@ internal PsesLogger(Logger logger, LogLevel minimumLogLevel)
                 ErrorLevelName, callerSourceFile, callerName, callerLineNumber, errorMessage, indentedException);
         }
 
+        /// <summary>
+        /// Log an exception that has been handled cleanly or is otherwise not expected to cause problems in the logs.
+        /// </summary>
+        /// <param name="errorMessage">The error message of the exception to be logged.</param>
+        /// <param name="exception">The exception itself that has been thrown.</param>
+        /// <param name="callerName">The name of the method in which the ILogger is being called.</param>
+        /// <param name="callerSourceFile">The name of the source file in which the ILogger is being called.</param>
+        /// <param name="callerLineNumber">The line number in the file where the ILogger is being called.</param>
+        public void WriteHandledException(
+            string errorMessage,
+            Exception exception,
+            [CallerMemberName] string callerName = null,
+            [CallerFilePath] string callerSourceFile = null,
+            [CallerLineNumber] int callerLineNumber = 0)
+        {
+            string indentedException = IndentMsg(exception.ToString());
+
+            _logger.Warning("[{WarningLevelName:l}] {CallerSourceFile:l}: In method '{CallerName:l}', line {CallerLineNumber}: Handled exception {ErrorMessage:l}{IndentedException:l}",
```

Comment:
Can we use messageTemplate here (and in WriteException) so we can standardize the templates.  Also move them to class fields, next to one another, so it obvious when one changes they all may need to be updated.  Part of that standardization (based on the current Write method) is to include the thread id, remove `method` and lower-case `In`. 

This is what I propose for the template for this method:
```
private readonly string _handledExceptionMessageTemplate = 
    "[{WarningLevelName:l}] tid:{threadId} in '{CallerName:l}' {CallerSourceFile:l}:{CallerLineNumber}: {IndentedLogMsg:l}"; 

…

string indentedLogMsg = IndentMsg(String.Join("Handled exception: ", errorMessage, Environment.NewLine, exception.ToString())); 

_logger.Warning(_handledExceptionMessageTemplate , WarningLevelName, callerName, callerSourceFile, callerLineNumber, indentedLogMsg);
```

Reply:
I assume that string templating works. The only reason I didn't already do stuff like this is because Serilog has these strangely magical string templates.

Reply:
It seems to work fine for the `Write` method.  :-)

Reply:
I'll make the changes this week :)